src/proxy_connect/index.jsx

```diff
@@ -1,7 +1,13 @@
 import { connect } from 'react-redux';
 import { getDisplayName, getForwardedComponent } from '../utils';
 
-
+/**
+ * @param mapStateToProps
+ * @param mapDispatchToProps
+ * @param mergeProps
+ * @param opts
+ * @return { React.Component } Wrapped with Redux.Connect component
```

Comment:
does it matter if there is a space after and before the brackets?

Reply:
nope :( , on generated page it will be without spaces anyway